shared-libs/outbound/src/outbound.js

```diff
@@ -177,15 +179,39 @@ const sendPayload = (payload, config) => {
   });
 };
 
-const updateInfo = (recordInfo, configName) => {
+// Never change this hashing algorithm or how we stringify, otherwise you will invalidate all existing hashes
+const hash = payload => crypto.createHash('sha256').update(JSON.stringify(payload)).digest('hex');
```

Comment:
We can probably bikeshed this if we'd like, but the general thought process here is:
 - Use crypto because it comes with node (so no CRC)
 - Pick something popular and not esoteric so it's impossible to not have it on your OpenSSH/OpenLibre implementation, and also so people don't wonder why you picked it
 - We're not using this for anything actually crypto related, so the security of the algo doesn't matter
 - But not md5 because it still makes my skin crawl
 - sha256 is as fast and mainstream as md5

Reply:
I'm happy with this - no need to bikeshed it.

Do you think it's worth adding this as a comment to the code?